pkg/project/resource_project_repo.go

```diff
@@ -153,13 +154,36 @@ var deleteRepos = func(ctx context.Context, projectKey string, repoKeys []RepoKe
 var deleteRepo = func(ctx context.Context, projectKey string, repoKey RepoKey, m interface{}) error {
 	tflog.Debug(ctx, fmt.Sprintf("deleteRepo: %s", repoKey))
 
-	_, err := m.(*resty.Client).
+	type Error struct {
+		Code    string `json:"code"`
+		Message string `json:"message"`
+	}
+
+	type ErrorResponse struct {
+		Errors []Error `json:"errors"`
+	}
+
+	var errorResp ErrorResponse
+
+	resp, err := m.(*resty.Client).
 		R().
 		AddRetryCondition(retryOnSpecificMsgBody("A timeout occurred")).
 		AddRetryCondition(retryOnSpecificMsgBody("Web server is down")).
 		AddRetryCondition(retryOnSpecificMsgBody("Web server is returning an unknown error")).
 		SetPathParam("repoKey", string(repoKey)).
+		SetError(&errorResp).
```

Comment:
Don't we need to instantiate `errorResp` before passing it to `SetError`?

Reply:
Nope. `var` will initialize it (and all its fields, recursively) with default values.